science/cdf-to-fits/Portfile

```diff
@@ -1,44 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem              1.0
 
 name                    cdf-to-fits
-version                 1.0
-revision                1
+version                 1.0-20180227
 maintainers             {aronnax @lpsinger}
```

Comment:
Please feel free to add yourself as a co-maintainer.

Reply:
Oh I don't know about that, I just wanted the thing to build! 😊

Reply:
Could we make it openmaintainer?